### ISSUE TITLE
Provide automatic conversion of type and enum imports

### ISSUE THREAD

**Issue by busunkim96** [googleapis/gapic-generator-python#266]
It would be nice if the conversion could be done automatically or if enums and types could continue to be available at the previous location `google.cloud.texttospeech.types` and `google.cloud.texttospeech.enums`.

Monocode
```py
from google.cloud import texttospeech
 
voice = texttospeech.types.VoiceSelectionParams(
        language_code='en-US',
        ssml_gender=texttospeech.enums.SsmlVoiceGender.NEUTRAL)
```

Microcode
```py
from google.cloud import texttospeech

voice = texttospeech.VoiceSelectionParams(
        language_code='en-US',
        ssml_gender=texttospeech.SsmlVoiceGender.NEUTRAL)
```

[This PR](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/2698/files) has more examples.

**Comment by lukesneeringer** [comment]
I thought I made the enums available on `texttospeech.SsmvlVoiceGender.NEUTRAL`?

**Comment by busunkim96** [comment]
@lukesneeringer Whoops, I copied-pasted the same code twice. :sweat_smile: It's fixed now.

In the microgenerator the enums and types are available under `texttospeech` which is much nicer than having to do `texttospeech.enums` and `texttospeech.types`.  But when folks are switching over they'll have to manually change the import paths. 

![image](https://user-images.githubusercontent.com/8822365/72190493-8105eb00-33b4-11ea-8620-977d9398966c.png)


I was wondering if it would be technically feasible to either:

1. Provide a script to fix the imports automatically.
2. Continue to provide enums and types at `texttospeech.enums` and `texttospeech.types`.

**Comment by software-dov** [comment]
I'm not sure I fully understand the description: is it that
```python
from google.cloud import texttospeech

ssml_gender = texttospeech.enums.SsmlVoiceGender.NEUTRAL
```
is no longer valid because there is no enum submodule in `texttospeech`?

**Comment by busunkim96** [comment]
@software-dov Yep, that's it.

**Comment by software-dov** [comment]
Thanks for clarifying. I'll get right on that.

**Comment by software-dov** [comment]
Technically a bugfix in that it's a gratuitous delta in the exposed surface.

**Comment by software-dov** [comment]
Fixed by #273
